prometheus2/prometheus2.spec

```diff
@@ -1,9 +1,9 @@
 %define debug_package %{nil}
 
 Name:		 prometheus2
-Version: 2.0.0
+Version: 2.1.0
 Release: 2%{?dist}
```

Comment:
Could you please change this to `Release: 1%{?dist}` since we have a new version number and the release number can be reset?